kornia/geometry/linalg.py

```diff
@@ -229,36 +228,18 @@ def point_line_distance(point: Tensor, line: Tensor, eps: float = 1e-9) -> Tenso
 
 def batched_dot_product(x: Tensor, y: Tensor, keepdim: bool = False) -> Tensor:
```

Comment:
Need's to add a `.data` at kornia/geometry/line.py:144 and 150

Reply:
i think we should start adopting here `wrap` / `unwrap`